### ISSUE TITLE
This repo should be Awesome!

### ISSUE THREAD

**Issue by thomasdom** [friendsofsylius/SyliusGoose#26]
I think this repository should be [Awesome](https://github.com/sindresorhus/awesome) to gain popularity and to let more OSS contributors and other developers know that Sylius exists!

**Comment by Holicz** [comment]
I agree. I was searching for this repo and could not find it. If it was Awesome I would find it in a second, because it's a kind of standard.

**Comment by gabiudrescu** [comment]
@bitbager let's rename it to FriendsOfSylius/Awesome - wdyt?

**Comment by thomasdom** [comment]
To respect the naming convention of Awesome repositories, you should name it "FriendsOfSylius/awesome-sylius".
